src/libmongoc/src/mongoc/mongoc-server-api.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "mongoc-config.h"
+#include "mongoc-server-api-private.h"
+
+const char *
+mongoc_server_api_version_to_string (mongoc_server_api_version_t version)
+{
+   if (version == MONGOC_SERVER_API_V1) {
+      return "1";
+   } else {
+      return NULL;
+   }
```

Comment:
Using a `switch` here would ensure that all `enum` values are covered:

```suggestion
   switch (version) {
      case MONGOC_SERVER_API_V1:
         return "1";
   }
   
   return NULL;
```

This also fixes the nit of not liking `return` nested in `else` as it may just be folded out of the conditional.